With current search text hint enabled, word and line mode key tags are not displayed

If I enable the checkbox that enables the current search text hint, then AceJump line mode or word mode is displayed without the tag keys. Disabling the checkbox makes the tags work as expected.

![image](https://user-images.githubusercontent.com/1888526/118521112-5fa0d600-b743-11eb-87ac-ecdd3cde2d11.png)
![image](https://user-images.githubusercontent.com/1888526/118521135-662f4d80-b743-11eb-9e38-f75fa30dbbe4.png)

The IDE also reports this exception:

```stacktrace
java.lang.AssertionError: INHERITED_COLOR_MARKER
	at com.intellij.ui.ColorUtil.lambda$marker$0(ColorUtil.java:24)
	at com.intellij.ui.JBColor.getColor(JBColor.java:135)
	at com.intellij.ui.JBColor.getRGB(JBColor.java:164)
	at java.desktop/java.awt.Color.equals(Color.java:699)
	at java.desktop/javax.swing.JComponent.setBackground(JComponent.java:2756)
	at org.acejump.view.TextHighlighter.showSearchNotification(TextHighlighter.kt:122)
	at org.acejump.view.TextHighlighter.render(TextHighlighter.kt:81)
	at org.acejump.session.Session.updateSearch(Session.kt:113)
	at org.acejump.session.Session.updateSearch$default(Session.kt:105)
	at org.acejump.session.Session.startRegexSearch(Session.kt:188)
	at org.acejump.session.Session.startRegexSearch(Session.kt:198)
	at org.acejump.action.AceAction$BaseRegexSearchAction.invoke(AceAction.kt:58)
	at org.acejump.action.AceAction.actionPerformed(AceAction.kt:35)
	at com.intellij.openapi.actionSystem.ex.ActionUtil.lambda$performActionDumbAware$5(ActionUtil.java:273)
	at com.intellij.util.SlowOperations.lambda$allowSlowOperations$0(SlowOperations.java:77)
	at com.intellij.util.SlowOperations.allowSlowOperations(SlowOperations.java:64)
	at com.intellij.util.SlowOperations.allowSlowOperations(SlowOperations.java:76)
	at com.intellij.openapi.actionSystem.ex.ActionUtil.performActionDumbAware(ActionUtil.java:273)
	at com.intellij.openapi.keymap.impl.IdeKeyEventDispatcher$1.performAction(IdeKeyEventDispatcher.java:616)
	at com.intellij.openapi.keymap.impl.IdeKeyEventDispatcher.lambda$processAction$3(IdeKeyEventDispatcher.java:676)
	at com.intellij.openapi.application.TransactionGuardImpl.performUserActivity(TransactionGuardImpl.java:95)
	at com.intellij.openapi.keymap.impl.IdeKeyEventDispatcher.processAction(IdeKeyEventDispatcher.java:675)
	at com.intellij.openapi.keymap.impl.IdeKeyEventDispatcher.processAction(IdeKeyEventDispatcher.java:626)
	at com.intellij.openapi.keymap.impl.IdeKeyEventDispatcher.processActionOrWaitSecondStroke(IdeKeyEventDispatcher.java:486)
	at com.intellij.openapi.keymap.impl.IdeKeyEventDispatcher.inInitState(IdeKeyEventDispatcher.java:481)
	at com.intellij.openapi.keymap.impl.IdeKeyEventDispatcher.dispatchKeyEvent(IdeKeyEventDispatcher.java:232)
	at com.intellij.ide.IdeEventQueue.dispatchKeyEvent(IdeEventQueue.java:889)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:833)
	at com.intellij.ide.IdeEventQueue.lambda$dispatchEvent$8(IdeEventQueue.java:449)
	at com.intellij.openapi.progress.impl.CoreProgressManager.computePrioritized(CoreProgressManager.java:808)
	at com.intellij.ide.IdeEventQueue.lambda$dispatchEvent$9(IdeEventQueue.java:448)
	at com.intellij.openapi.application.impl.ApplicationImpl.runIntendedWriteActionOnCurrentThread(ApplicationImpl.java:781)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:502)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:203)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:124)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:113)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:109)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:90)
```

--- a/src/main/kotlin/org/acejump/view/TextHighlighter.kt
+++ b/src/main/kotlin/org/acejump/view/TextHighlighter.kt
@@ -117,9 +117,13 @@ internal class TextHighlighter {
       panel.add(label1, BorderLayout.WEST)
       panel.add(label2, BorderLayout.CENTER)
       panel.add(label3, BorderLayout.EAST)
-      panel.border = BorderFactory.createLineBorder(jumpMode.caretColor)
 
-      panel.background = jumpMode.caretColor
+      if (jumpMode == JumpMode.DISABLED) {
+        panel.border = BorderFactory.createLineBorder(Color.BLACK)
+      } else {
+        panel.border = BorderFactory.createLineBorder(jumpMode.caretColor)
+      }
+
       panel.preferredSize = Dimension(it.contentComponent.width +
           label1.preferredSize.width, panel.preferredSize.height)
 

